src/domains/idpay-app/api/idpay_wallet/openapi.wallet.yml.tpl

```diff
@@ -0,0 +1,689 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+              example:
+                code: 0
+                message: string
+        '429':
+          description: Too many Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+              example:
+                code: 0
+                message: string
+        '500':
+          description: Server ERROR
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+              example:
+                code: 0
+                message: string
+  /iban:
+    put:
+      tags:
+        - wallet
+      summary: Association of an IBAN to an initiative
+      operationId: enrollIban
+      requestBody:
+        description: 'Unique identifier of the subscribed initiative, IBAN of the citizen'
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/IbanPutDTO'
+            example:
+              initiativeId: string
+              iban: string
+              description: string
+      responses:
+        '200':
+          description: Enrollment OK
+          content:
+            application/json: { }
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+              example:
+                code: 0
+                message: string
+        '401':
+          description: Authentication failed
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+              example:
+                code: 0
+                message: string
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+              example:
+                code: 0
+                message: string
+        '404':
+          description: The requested ID was not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+              example:
+                code: 0
+                message: string
+        '429':
+          description: Too many Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+              example:
+                code: 0
+                message: string
+        '500':
+          description: Server ERROR
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDTO'
+              example:
+                code: 0
+                message: string
+  /instrument:
```

Comment:
Here is better to have "/{initiativeId}/instruments" in the path as in the operation below and remove the initiativeId from the InstrumentPutDTO.

Reply:
Changed in commit -> https://github.com/pagopa/cstar-infrastructure/pull/452/commits/0cdc4755001ccdfd71568141942af305e0be5714